frontend/src/components/BookCard.tsx

```diff
@@ -0,0 +1,26 @@
+import {Book} from "../models/Book";
+
+type Props = {
```

Comment:
I would aggree to Lena :-)
```suggestion
type BookCardProps = {
```